js/apps/system/_admin/aardvark/APP/react/src/utils/constants.ts

```diff
@@ -23,8 +23,13 @@ export type DispatchArgs<FormState> = {
   formState?: FormState;
 };
 
+export type RenderView = "LinkList" | "AddNew" | "ViewParent" | "ViewChild" | "ViewField";
+
 export type FormProps<FormState> = {
   formState: FormState;
   dispatch: Dispatch<DispatchArgs<FormState>>;
   disabled?: boolean;
+  view?: string;
+  show?: RenderView;
```

Comment:
This should be an extension to base form state that is views-specific. The common `utils/constants` should not contain anything that only applies to views.

Reply:
Done.